src/winbind.c

```diff
@@ -206,15 +206,78 @@ uint32_t winbind_cli_auth(char *user, char *domain,
     return ret;
 }
 
+const char gssntlmssp_sids_urn[] = "urn:gssntlmssp:sids";
+
+static uint32_t format_sids_as_name_attribute(
```

Comment:
Break before the function name as you did above.

Reply:
But for the case above you said 'spurious indentation', no?
Is it what you want (I prefer you show me how exactly you see it, keeping 80 characters length limit in mind):
```
static uint32_t
    format_sids_as_name_attribute(const struct wbcAuthUserInfo *wbc_info,
                                  struct gssntlm_name_attribute **auth_attrs)
```

Reply:
please see the style I mentioned above if the first line with argument ends up being too long.
do not indent the function name on its own line. that's not the style we use in gssntlmssp

Reply:
Sure, will do.
But still waiting for use-case like this:
`const struct veeeeeeeeeeeery_long_struct_name *veeeeeeeeeeeeeeeeery_long_func_name(...)`
I know few people who very likes self-explanatory but very long names :-)

Reply:
We do not like very long names, problem solved :-)